server/lib/search.js

```diff
@@ -92,7 +92,7 @@ const sanitizeSearchTermForILike = term => {
   return term.replace(/(_|%|\\)/g, '\\$1');
 };
 
-const getSearchTermSQLConditions = term => {
+const getSearchTermSQLConditions = (term, collectiveTable) => {
```

Comment:
@Betree : After the rebase I've added a new arg to this function as when we use it within the `searchCollectivesInDB` function we need to refer to to the collective table.